test/sql/json/test_json_extract.test

```diff
@@ -0,0 +1,193 @@
+# name: test/sql/json/test_json_extract.test
+# description: Test JSON extract
+# group: [json]
+
+require json
+
+statement ok
```

Comment:
Perhaps we could have a look at converting the [SQLite JSON tests](https://github.com/sqlite/sqlite/blob/master/test/json101.test) (sadly not in SQLLogicTest format), especially as we add more of the SQLite JSON functionality.

Reply:
I've added some of the tests, will add more as I add more functionality